CITATION

```diff
@@ -1,6 +1,8 @@
-@unpublished{spacy2,
-    AUTHOR = {Honnibal, Matthew and Montani, Ines},
-    TITLE  = {{spaCy 2}: Natural language understanding with {B}loom embeddings, convolutional neural networks and incremental parsing},
-    YEAR   = {2017},
-    Note   = {To appear}
+@software{spacy,
+  author = {Honnibal, Matthew and Montani, Ines},
+  title = {{spaCy: Natural language understanding with Bloom embeddings, convolutional neural networks and incremental parsing}},
```

Comment:
If we're updating the citation anyways, we might as well update the title? I think this would be most consistent:

```suggestion
  title = {{spaCy: Industrial-strength Natural Language Processing in Python}},
```